src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -93,278 +94,396 @@ public void Trace(Transaction transaction, BlockHeader block, ITxTracer txTracer
             Execute(transaction, block, txTracer, ExecutionOptions.NoValidation);
         }
 
-        private void QuickFail(Transaction tx, BlockHeader block, ITxTracer txTracer, bool eip658NotEnabled,
-            string? reason)
+        protected virtual void Execute(Transaction tx, BlockHeader blk, ITxTracer tracer, ExecutionOptions opts)
```

Comment:
don't like blk :D for blockHeader Why not blockHeader or just header?

Reply:
done